climpred/tests/conftest.py

```diff
@@ -30,8 +30,8 @@ def PM_da_initialized_1d_lead0(PM_da_initialized_1d):
     framework."""
     da = PM_da_initialized_1d
     # Convert to lead zero for testing
-    da['lead'] -= 1
-    da['init'] += 1
+    da['lead'] = da['lead'] + 1
+    da['init'] = da['init'] - 1
```

Comment:
I think the plus and minus are backwards here and should be fixed. Lead goes back 1 to start at zero; init goes forward one since lead was modified to start at zero.

Reply:
thanks for finding this. however the tests didnt find this!